### DISCUSSION TITLE
how to not logout out of google accounts

### DISCUSSION THREAD

**Discussion by ivanjx** [datalust/seq-tickets#2268] [Help] [answered]
i have setup oidc login with google. it works perfectly but if i press logout from seq, it also logs me out of all of my google accounts. how to prevent seq from logging me out from google?

**Comment by nblumhardt** [comment, marked as answer]
Hi @ivanjx - thanks for your message. Do you have a logout URI explicitly set in Settings > System > Authentication (change)? If not, specifying a benign one might provide a way around this. There's no explicit customization on the Seq side right now, though. Hope this helps!

**Reply by ivanjx** [reply]
> Do you have a logout URI explicitly set

no. im not sure what would be the logout uri for google's oauth. and yes setting the logout uri to other sites works instead of leaving it just empty.
